{"issue_url": "https://api.github.com/repos/rails/rails/issues/9119", "repo": "rails", "body": "Thanks. What do you think about just [raising if the task cannot be found in `class_for_adapter`](https://github.com/dazuma/rails/blob/7f64b5aec219204ddb1b5e747ba732c65fa4db81/activerecord/lib/active_record/tasks/database_tasks.rb#L132)? Then we would not need the default class, only the exception, and the adapter option is already available there. Other than that, I think it's :+1:", "_id": {"$oid": "5237ecddbd3543c151009d53"}, "issue_id": 9119, "url": "https://api.github.com/repos/rails/rails/issues/comments/12885352", "html_url": "https://github.com/rails/rails/pull/9119#issuecomment-12885352", "updated_at": "2013-01-30T11:27:18Z", "user": {"subscriptions_url": "https://api.github.com/users/carlosantoniodasilva/subscriptions", "events_url": "https://api.github.com/users/carlosantoniodasilva/events{/privacy}", "repos_url": "https://api.github.com/users/carlosantoniodasilva/repos", "gists_url": "https://api.github.com/users/carlosantoniodasilva/gists{/gist_id}", "url": "https://api.github.com/users/carlosantoniodasilva", "login": "carlosantoniodasilva", "gravatar_id": "77237b97a465ae5a293ad323b7296837", "html_url": "https://github.com/carlosantoniodasilva", "following_url": "https://api.github.com/users/carlosantoniodasilva/following{/other_user}", "received_events_url": "https://api.github.com/users/carlosantoniodasilva/received_events", "organizations_url": "https://api.github.com/users/carlosantoniodasilva/orgs", "avatar_url": "https://0.gravatar.com/avatar/77237b97a465ae5a293ad323b7296837?d=https%3A%2F%2Fidenticons.github.com%2F60b997fd68261ce70cadaf6c8226c042.png", "starred_url": "https://api.github.com/users/carlosantoniodasilva/starred{/owner}{/repo}", "id": 26328, "followers_url": "https://api.github.com/users/carlosantoniodasilva/followers", "type": "User"}, "created_at": "2013-01-30T11:27:18Z", "id": 12885352, "owner": "rails"}
